### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by valguss** [deployphp/deployer#3695] [General]
Hi Guys. Just wondering when the next release will be created? Been waiting for some features that are merged for awhile.

**Comment by antonmedv** [comment]
Will do

**Reply by valguss** [reply]
Hi @antonmedv Any news on this. Is there a blocker I can help with?

**Reply by valguss** [reply]
Any update at all? been waiting an age for a new release

**Reply by antonmedv** [reply]
🚀 Attention, Deployer enthusiasts! 🎉 The long-awaited new version is ready to launch. However, it's currently hovering in 'orbit', waiting for a small boost. Here's the deal: if we can gather a 'mission fund' of $100 in donations today, I'll hit the 'launch' button faster than you can say 'deployment'. Let's fuel this rocket! 🚀💸 Remember, it's all in the spirit of open-source fun (and to keep me stocked with enough coffee to code without sleep! ☕😉). Let's make this launch a memorable one! 🌟
